assets/app/scripts/services/data.js

```diff
@@ -504,14 +504,48 @@ angular.module('openshiftConsole')
     }
   };
 
-  DataService.prototype._watchWebsocketRetries = function(type, context, retry) {
+  DataService.prototype._addWatchWebsocketEvent = function(type, context, eventType) {
```

Comment:
I like the short name better. I was trying to be consistent with the other names, which all were `watchWebsockXYZ`. I'll make the change.